dev/com.ibm.ws.security.authentication.filter_fat/.classpath

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="/build.sharedResources/lib/junit/old/junit.jar">
```

Comment:
This should be pulled in from the bnd.bnd file, not defined here.

Reply:
Of course this PR was already merged.  I made a follow on PR to fix up the .classpath files so that things built correctly in eclipse.